frontend/src/pull-card/index.tsx

```diff
@@ -15,6 +16,12 @@ const Card = chakra(Flex, {
       "&:hover .build_status": {
          opacity: 1
       },
+      "& .copy": {
+         visibility: "hidden",
```

Comment:
How come we are hiding the copy icon unless we hover over it? Shouldn't we always show elements that can be interacted with? 

![Interactability](https://user-images.githubusercontent.com/22064420/159299904-c69a85a7-fb3e-470f-be1c-6022cc6b5e8c.gif)

Reply:
Ahhhh is this to not clutter the non-focused cards?

Reply:
Correct.